Documentation/policy/language.rst

```diff
@@ -1032,6 +944,97 @@ allowed but connections to the returned IPs are not, as there is no L3
           ``servicename.namespace.svc.cluster.local.`` must have the latter
           allowed with ``matchName`` or ``matchPattern``.
 
+Obtaining DNS Data for use by ``toFQDNs``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+IPs are obtained via intercepting DNS requests with a proxy or DNS polling, and
+matching names are inserted irrespective of how the data is obtained. These IPs
+can be selected with ``toFQDN`` rules. DNS responses are cached within cilium
+agent respecting TTL.  
+
+.. _DNS Proxy:
+
+DNS Proxy
```

Comment:
I was worried this might end up putting `(preferred)` in the middle of sentences everywhere, but it appears not. I think it's useful to highlight that this approach is more robust than the other, though.